src/Component/Symbolizer/Field/ColorField/ColorField.example.md

```diff
@@ -36,12 +36,11 @@ import { ColorField } from 'geostyler';
 
 const ColorFieldExample = () => {
 
-  const [value, setValue] = useState();
+  const [value, setValue] = useState("#013370");
```

Comment:
Will this lead to the same behavior as before?

Reply:
This is just the example code. The behaviour is visible in the gif.

Reply:
Good point :grinning: